firstvoices/backend/models/jobs.py

```diff
@@ -50,13 +50,11 @@ class Meta:
             models.Index(fields=["site", "is_preview"], name="corr_site_preview_idx"),
         ]
 
-    latest_recalculation_date = models.DateTimeField(auto_now_add=True)
-    latest_recalculation_result = models.JSONField()
-    task_id = models.CharField(max_length=255, null=True, blank=True)
+    cleanup_result = models.JSONField(blank=True, null=True)
     is_preview = models.BooleanField(default=True)
 
     def __str__(self):
-        return self.site.title + " - " + str(self.latest_recalculation_date)
+        return self.site.title + " - DictionaryCleanup - " + self.status
```

Comment:
I think this __str__ needs something more individual than status, probably just the id. Like `self.site.title DictionaryCleanupJob self.id`